pkg/controllers/proxy/network_services_controller.go

```diff
@@ -147,7 +147,7 @@ func (ln *linuxNetworking) ipAddrDel(iface netlink.Link, ip string) error {
 	if err == nil {
 		out, err := exec.Command("ip", "route", "delete", "local", ip, "dev", KubeDummyIf,
 			"table", "local", "proto", "kernel", "scope", "host", "src",
-			NodeIP.String(), "table", "local").CombinedOutput()
+			ln.nodeIPv4.String(), "table", "local").CombinedOutput()
```

Comment:
I wonder if there should also be one for ln.nodeIPv6 pointing at the ip -6 route ....